### PR TITLE
[release-v2.10] migrating to self-hosted runners: pull-request.yaml jobs

### PR DESCRIPTION
**Pull request by nicholasSUSE**
### Issue: https://github.com/rancher/ecm-distro-tools/issues/491

Migrating to self-hosted runners and testing

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    name: Validate
+    runs-on: org-${{ github.repository_owner_id }}-amd64-k8s
+    container: registry.suse.com/bci/bci-base:latest
+    steps:
+      - name: install dependencies
+        run: zypper --non-interactive install docker jq git make go awk
+      - name: install gh
+        run: |
+          mkdir -p /tmp/gh
+          curl -fsL https://github.com/cli/cli/releases/download/v2.63.2/gh_2.63.2_linux_amd64.tar.gz | tar xvzf - --strip-components=1 -C /tmp/gh
```
Comment (tashima42) [inline .github/workflows/build.yaml]: Can you separate the CLI version into a variable?

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          else
+            echo "Validating local branch"
+            make validate
+          fi
+
+      - name: Check container images
+        run: make check-images
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```
Comment (tashima42) [inline .github/workflows/build.yaml:88]: Are these creds still available through secrets on this repo?

Comment (nicholasSUSE) [reply .github/workflows/build.yaml:88]: Yes they are!

They should not be?

Comment (tashima42) [reply .github/workflows/build.yaml:88]: Other repos are using the EIO vault, but if the secrets are available it's fine.

**Comment by nicholasSUSE** [comment]
closing because I am pushing from a fork

**Pull request closed without merging**